### DISCUSSION TITLE
Fetch dashboard id/user id/tanent id within the plugin

### DISCUSSION THREAD

**Discussion by shuotli** [grafana/grafana#39376] [Plugins]
Is there a way to fetch dashboard id/user id/tanent id within the plugin? We are working on logging telemetry with App Insights.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
